multiple_generate.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+while getopts s:e:u: option
+do
+ case "${option}"
+ in
+ s) SUBPROJECT=${OPTARG};;
+ e) EMAIL=${OPTARG};;
+ u) UNIQUEID=${OPTARG};;
+ esac
+done
+
+function clean() {
+  rm -rf $BUILD_DIR
+}
+
+BASE=$(pwd)
+
+if [ "${WEBUI:-false}" == "true" ]; then
+  BUILD_DIR=$(pwd)/temp/$EMAIL/$UNIQUEID
+  mkdir -p $BUILD_DIR
+  cd $BUILD_DIR
+else
+  git clone https://github.com/fossasia/yaydoc.git yaydocclone
+  BASE=$(pwd)/yaydocclone
+  BUILD_DIR=$(pwd)/temp
+  echo -e "build dir"
+  echo -e $BUILD_DIR
+  mkdir -p $BUILD_DIR
+  cd $BUILD_DIR
+fi
+
+
+IFS=', ' read -r -a SUBPROJECTS <<< "$SUBPROJECT"
```

Comment:
I think the best way would be that we split with a ',' and then strip whitespaces from both ends.